sphinxcontrib/phpdomain.py

```diff
@@ -22,7 +22,7 @@
 from sphinx.util.nodes import make_refnode
 from sphinx.util.compat import Directive
 from sphinx.util.docfields import Field, GroupedField, TypedField
-
+from sphinx.domains.python import _pseudo_parse_arglist
```

Comment:
> Copy, not import, and use this function

That sounds like a good option to me. Better copy the code vs. have the code break in the future when sphinx changes their implementation.